doc/faq/howto_faq.rst

```diff
@@ -233,7 +232,7 @@ Automatically make room for tick labels
 In most use cases, it is enough to simply change the subplots adjust
 parameters as described in :ref:`howto-subplots-adjust`.  But in some
 cases, you don't know ahead of time what your tick labels will be, or
-how large they will be (data and labels outside your control may be
+how large they will be (data and labels outside your control maybe
 being fed into your graphing application), and you may need to
```

Comment:
In either case, the wording here is awkward. "may be being" and "maybe being" are both poor wording, so this should just get reworded entirely, I think.